src/nft/hooks/useSellAsset.ts

```diff
@@ -23,6 +25,12 @@ export const useSellAsset = create<SellAssetState>()(
       sellAssets: [],
       selectSellAsset: (asset) =>
         set(({ sellAssets }) => {
+          const trace = useTrace()
```

Comment:
I am currently unable to select items to add to bag in the preview

Reply:
Elsewhere I believe we have the event triggering on the button press instead of the hook, we may want to move this event there to maintain consistency

Reply:
you're right thanks for catching yikes ... shouldn't be using a hook here for trace. I moved it to the button components.